ext/tags.go

```diff
@@ -88,6 +88,18 @@ var (
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
+
+	//////////////////////////////////////////////////////////////////////
+	// PerfEvent Tag
+	//////////////////////////////////////////////////////////////////////
+
+	// PerfEvent records the list of events given in a comma separated string (e.g.
+	// "cpu-cycles,cache-misses,branch-misses") for a span.
+	// The tracer is expected to start the events in PerfEvent using the "perfevents"
+	// package (https://github.com/opentracing-contrib/perfevents/go) . On span
+	// Finish(), the events must be closed.
+
+	PerfEvent = stringTagName("perfevents")
```

Comment:
would it perhaps make more sense to create a `Tracer` that wraps a given "real" `Tracer` and provides the perfevents for each span therein?